source/addComment.js

```diff
@@ -1,3 +1,32 @@
-$(document).ready(function(){
-  // add your code here
-})
+/*jshint esversion: 6 */
+
+$('button').on('click', function() {
+  $('button').hide();
+
+  $('body').append('<form id=new_comment name="new_comment" style="display: block"></form>');
+
+  var commentInput = '<textarea name="comment_input" placeholder="Your comment here..."></textarea>';
```

Comment:
With ES6, always start with `const`, and change to `let` if you get an error about reassigning the `const`. `var` kind of goes out the door.